templates/web/base/admin/category-checkboxes.html

```diff
@@ -22,4 +14,23 @@
       </li>
     [% END %]
   </ul>
+[% END %]
+<fieldset>
+  <legend>
+  [% IF hint %]
+    <div class="admin-hint">
+      <p>[% hint %]</p>
+    </div>
+  [% END %]
+    [% loc('Categories:') %]
+  </legend>
+
+  [% IF category_groups %]
+    [% FOR group IN category_groups %]
+      <h3>[% group.name | html %]</h3>
+      [% INCLUDE checkboxes contacts=group.categories %]
```

Comment:
If a category is in multiple groups it will now be listed under all of them. Presumably if you want to deselect that means you'd have to untick it from all of them? I realise this won't be that likely, but only listing them under one group and have some extra text saying "also in a, b, c" might be better.

Reply:
Good idea. I've put them in a "Multiple Groups" pseudogroup at the bottom of the list with a note as you suggest.